src/content/app/genome-browser/components/browser-nav-button/browserNavConfig.ts

```diff
@@ -34,20 +32,20 @@ export type BrowserNavItem = {
 
 export const browserNavConfig: BrowserNavItem[] = [
```

Comment:
I've been wondering why do we even need this file. Why not write most of this inline in jsx in the component responsible for rendering the nav buttons?

Reply:
✅